spec/rdkafka/producer_spec.rb

```diff
@@ -21,38 +21,73 @@
       expect(producer.delivery_callback).to be_a Proc
```

Comment:
Could we change this line to `expect(producer.delivery_callback).to respond_to :call` as we do in the `config_spec.rb` above, and add a test for a callable object?

Also, while you're here, would you mind changing the `puts stats` on line 18 to be `puts delivery_handle`?  Even though the code will never get executed, we probably should have something that would execute if it were asked to. :)

Reply:
Heh, a little copy pasta 🍝. Updated the `puts` and added a spec for setting the callback.